### DISCUSSION TITLE
How to make EDDI voice output in other languages

### DISCUSSION THREAD

**Discussion by AloneKnightBole** [EDCD/EDDI#2320] [EDDI Standalone Q&A] [answered]
I was amazed at the power of EDDI，but my native language is Chinese, I want to hear Chinese sounds.So I tried to change the language through the ‘Edit’ function in ‘speech responder‘.I managed to do some of that.But there are still parts that cannot be translated.So I wonder if this idea can be done and What would I do if I could.By the way, I am not a computer major, so my expression may not be accurate

**Comment by AloneKnightBole** [comment]
I just converted the language in Edit to Chinese and used TTS to speak another language, not English

**Comment by Tkael** [comment, marked as answer]
Hi @AloneKnightBole!
We need help from native speakers to translate EDDI into languages that we do not yet support. Please take a look at <https://github.com/EDCD/EDDI/blob/stable/docs/Localization_for_translators.md> if you are interested in helping create a Chinese translation. o7
